### ISSUE TITLE
矩形框和尺子的LableName可以展示在影像的矩形框旁边吗

### ISSUE THREAD

**Issue by niruiying** [Kitware/VolView#534]
矩形框和尺子的LableName可以展示在影像的矩形框旁边作为注释吗
![1703056774772](https://github.com/Kitware/VolView/assets/154293515/ef271ad0-fb81-4841-830a-1ed5ed3c6ac2)

**Comment by PaulHax** [comment]
Good idea.  Right now, right clicking on the rectangle shows the label.  

Right clicking is not so "discoverable."  For another approach, some sort of tooltip?  Likely would involve changing this file:
https://github.com/Kitware/VolView/blob/main/src/components/tools/rectangle/RectangleSVG2D.vue

**Comment by PaulHax** [comment]
When we have the select tool active, we show a tooltip when hovering on a vector annotaiton.  But we should show a tootip when we have the rectange/polgyon/ruler tool active too.
